code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,773 @@
+					/obj/item/reagent_containers/syringe/insulin = 1, /obj/item/reagent_containers/hypospray/autoinjector = 2, /obj/item/reagent_containers/food/pill/salbutamol = 2, /obj/item/reagent_containers/food/pill/mannitol = 2, /obj/item/reagent_containers/food/pill/mutadone = 1,
+					/obj/item/stack/medical/bruise_pack/advanced = 1, /obj/item/stack/medical/ointment/advanced = 1, /obj/item/stack/medical/bruise_pack = 2, /obj/item/stack/medical/splint = 2, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/reagent_containers/dropper = 1,
+					/obj/item/healthanalyzer = 1, /obj/item/healthupgrade = 1, /obj/item/reagent_containers/iv_bag/slime = 1)
+	contraband = list(/obj/item/reagent_containers/glass/bottle/sulfonal = 1, /obj/item/reagent_containers/glass/bottle/pancuronium = 1)
+
+/obj/machinery/autolathe/hacked
+	hacked = 1
+
+/obj/machinery/light/broken
+	active_power_usage = 0
+	icon_state = "tube-broken"
+	status = 2
+
+/obj/machinery/light/small/broken
+	active_power_usage = 0
+	icon_state = "bulb-broken"
+	status = 2
+
+/obj/machinery/door/poddoor/shutters/rusted
+	name = "Rusted Shutter"
+	desc = "Heavy duty shutters that has been warped and corroded by the elements. It does not seem it will take much to break it down by force."
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 15, "energy" = 15, "bomb" = 0, "bio" = 50, "rad" = 100, "fire" = 50, "acid" = 25)
+	damage_deflection = 0
+	max_integrity = 300
+	icon_state = "rusted_closed"
+
+/obj/machinery/door/poddoor/shutters/rusted/do_animate(animation)
+	switch(animation)
+		if("opening")
+			flick("rusted_opening", src)
+			playsound(src, 'sound/machines/blastdoor.ogg', 30, 1)
+		if("closing")
+			flick("rusted_closing", src)
+			playsound(src, 'sound/machines/blastdoor.ogg', 30, 1)
+
+/obj/machinery/door/poddoor/shutters/rusted/update_icon()
+	if(density)
+		icon_state = "rusted_closed"
+	else
+		icon_state = "rusted_open"
+
+//////// MECHA ////////
+
+/obj/mecha/combat/gygax/dark/rusted
+	desc = "This once lightweight exosuit, painted in a dark scheme, has not been spared by the elements. Exterior armor is heavily rusted and most equipment sockets have decayed beyond repair."
+	name = "Rusted Dark Gygax"
+	max_integrity = 200
+	deflect_chance = 10
+	armor = list(melee = 20, bullet = 20, laser = 25, energy = 20, bomb = 10, bio = 0, rad = 10, fire = 50, acid = 50)
+	max_temperature = 25000
+	leg_overload_coeff = 50
+	operation_req_access = list(ACCESS_AWAY01)
+	internal_damage_threshold = 25
+	max_equip = 0
+	destruction_sleep_duration = 2
+
+/obj/mecha/combat/gygax/dark/rusted/add_cell()
+	cell = new /obj/item/stock_parts/cell/high(src)
+
+/obj/mecha/combat/gygax/dark/rusted/onTransitZ()
+	do_sparks(3, 1, src)
+	take_damage(300, BRUTE)
+
+////// STRUCTURES //////
+
+/obj/structure/displaycase/telecrystal
```

Comment:
Same for this one. Other than the AWAY required access. People might mistake this display case currently as a normal telecrystal display case. Can you specify it a bit more? Or make a generic `displaycase/telecrystal` and then have a specific DBS version here